commercial/app/views/contentapi/itemV2.scala.html

```diff
@@ -30,7 +31,7 @@
 }{
 
     <div class="adverts__row">
-        @itemLargeCard(item, clickMacro)
+        @itemLargeCard(card, omnitureId)
```

Comment:
It gets passed in as part of the `targetUrl` of the card content.
That's built here: https://github.com/guardian/frontend/blob/kc-capi-card/common/app/common/commercial/CardContent.scala#L79